test/pages/Signup.test.jsx

```diff
@@ -49,8 +49,8 @@ describe('Signup Page', () => {
         }
       }
     };
-    const { getByText } = render(<MockSignupContexts session={sessionObj} />);
-    expect(getByText('Register for PASS', { exact: false })).not.toBeNull();
+    const { getByRole } = render(<MockSignupContexts session={sessionObj} />);
+    expect(getByRole('heading')).not.toBeNull();
```

Comment:
And this works because either component that can be rendered have a heading?  If that is true, is that too generic of a test?

Reply:
True, I should probably add a `name` option to this to make it more specific. Thanks!

Reply:
Got it in there 👍